src/Exception/API/XsollaAPIException.php

```diff
@@ -39,8 +39,8 @@ public static function fromBadResponse(BadResponseException $previous)
         $message = sprintf(
             static::$messageTemplate,
             $previous->getMessage(),
-            str($previous->getRequest()),
-            str($previous->getResponse())
+            Message::toString($previous->getRequest()),
```

Comment:
Changes in class `XsollaAPIException` could break in some old environments.

* Function `str()` was removed in `guzzlehttp/psr7:2.0`.
* Method `Message::toString` was added in `guzzlehttp/psr7:1.7.0`.

We can add a wrapper method to handle this case, like the one proposed in PR #97.

Reply:
Yes, it sounds good 👍